frontend/src/pages/Admin/EnterPhoneNumber/style.tsx

```diff
@@ -1,15 +1,16 @@
 import { styled } from 'styled-components';
 
 export const Title = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: center;
   position: relative;
-  width: 100vw;
-  height: 110px;
+  width: 100%;
+  height: 80px;
```

Comment:
what's difference between `vw` and `%`?

Reply:
i think `vw` based on browser size but `%` based on parent element size.

Reply:
ㄴ It's correct. That's what i want to say.